app/models/ticket_request.rb

```diff
@@ -47,76 +47,59 @@ class << self
     #   and then for each ticket request we return the primary user with the ticket request info,
     #   followed by one row per guest.
     def for_csv(event)
-      table = []
+      [].tap do |table|
+        event.ticket_requests.active.each do |ticket_request|
```

Comment:
I don’t know what an active or an inactive table is, but the CSV should essentially be a dump of the table.

Reply:
Active means either paid or awaiting payment. It does not include declined, refunded etc